default/props.conf

```diff
@@ -23,6 +23,26 @@ EVAL-signature = case(category=="AdvancedHunting-DeviceAlertEvents", 'properties
 EVAL-type = case(category=="AdvancedHunting-DeviceAlertEvents", "alert", 1==1, null())
 EVAL-vendor_account = case(category=="AdvancedHunting-DeviceAlertEvents", properties.tenantId, 1==1, null())
 
+# Email node
+EVAL-action = case(category=="AdvancedHunting-EmailEvents",case('properties.DeliveryAction'=="Delivered","delivered",'properties.DeliveryAction'=="Blocked","blocked",'properties.DeliveryAction'=="Junked","deleted",'properties.DeliveryAction'=="Replaced","quarantined",true(),"other"))
+EVAL-filter_action = case(category=="AdvancedHunting-EmailEvents",'properties.EmailActionPolicy')
+EVAL-internal_message_id = case(in(category,"AdvancedHunting-EmailEvents","AdvancedHunting-EmailAttachmentInfo"),'properties.NetworkMessageId')
+EVAL-message_id = case(category=="AdvancedHunting-EmailEvents",'properties.InternetMessageId')
+EVAL-message_info = case(category=="AdvancedHunting-EmailEvents",'properties.ConfidenceLevel')
+EVAL-recipient = case(in(category,"AdvancedHunting-EmailEvents","AdvancedHunting-EmailAttachmentInfo"),'properties.RecipientEmailAddress')
+EVAL-recipient_count =  case(in(category,"AdvancedHunting-EmailEvents","AdvancedHunting-EmailAttachmentInfo"),mvcount('properties.RecipientEmailAddress'))
+EVAL-recipient_domain = case(in(category,"AdvancedHunting-EmailEvents","AdvancedHunting-EmailAttachmentInfo"),replace('properties.RecipientEmailAddress',"[^@]+@(.*)","\1"))
+EVAL-signature = case(category=="AdvancedHunting-EmailEvents",if('properties.EmailAction'=="null","No action taken",'properties.EmailAction'))
+EVAL-signature_extra = case(category=="AdvancedHunting-EmailEvents",'properties.DetectionMethods')
+EVAL-signature_id = case(category=="AdvancedHunting-EmailEvents",'properties.EmailActionPolicyGuid')
+EVAL-src = case(category=="AdvancedHunting-EmailEvents",'properties.SenderIPv4')
+EVAL-src_user = case(in(category,"AdvancedHunting-EmailEvents","AdvancedHunting-EmailAttachmentInfo"),'properties.SenderFromAddress')
+EVAL-src_user_domain = case(in(category,"AdvancedHunting-EmailEvents","AdvancedHunting-EmailAttachmentInfo"),replace('properties.SenderFromAddress',"[^@]+@(.*)","\1"))
+EVAL-subject = case(category=="AdvancedHunting-EmailEvents",'properties.Subject')
+# Extra fields for tag=content
+EVAL-file_hash = case(category=="AdvancedHunting-EmailAttachmentInfo",'properties.SHA256')
```

Comment:
This may be OK, but may also need to be merged with https://github.com/splunk/TA-microsoft-365-defender-advanced-hunting-add-on/blob/master/default/props.conf#L42